### ISSUE TITLE
Warning thrown with matplotlib 2.0

### ISSUE THREAD

**Issue by ericmjl** [ResidentMario/missingno#20]
I'm using matplotlib 2.0, and I thought I'd just quickly report this warning message that shows up when I call `msno.matrix(dataframe)`:

```
/Users/ericmjl/anaconda/lib/python3.5/site-packages/missingno/missingno.py:250: MatplotlibDeprecationWarning: The set_axis_bgcolor function was deprecated in version 2.0. Use set_facecolor instead.
  ax1.set_axis_bgcolor((1, 1, 1))
```

It's probably a low-priority, mission-noncritical change, but just putting it here for the record. If I do have the time to get myself familiarized with the codebase, I might just put in a PR for it! :smile:

**Comment by ResidentMario** [comment]
2.0 changes a lot of the defaults and deprecates a lot of things, this may not be the only update I have to make.

**Comment by ResidentMario** [comment]
Addressed in 0.3.3.
